lib/config.ts

```diff
@@ -93,9 +93,23 @@ export interface Config {
   sauceKey?: string;
   /**
    * Use sauceAgent if you need customize agent for https connection to
```

Comment:
Sorry, this bugs me a little. "Use sauceAgent if you need a customized agent for the https connection..."